react-app/src/App.js

```diff
@@ -10,32 +12,39 @@ export function useUserLogged() {
 }
 
 function App() {
+    const history = useHistory()
 	const [userLoggedIn, setUserLoggedIn] = useState(null)
 
+	useEffect(() => {
+		authListener(signUserIn)
+	}, [])
+
+	function signUserIn(user) {
+		setUserLoggedIn(user)
+		if (user) {
+            history.replace('/')
+		}
```

Comment:
Looks like some issues with auto indent? Or maybe it is Github's problem?
I see like 8 spaces for one indent level :thinking: